phpBB/phpbb/db/tools/doctrine.php

```diff
@@ -0,0 +1,394 @@
+<?php
+/**
+ *
+ * This file is part of the phpBB Forum Software package.
+ *
+ * @copyright (c) phpBB Limited <https://www.phpbb.com>
+ * @license GNU General Public License, version 2 (GPL-2.0)
+ *
+ * For full copyright and license information, please see
+ * the docs/CREDITS.txt file.
+ *
+ */
+
+namespace phpbb\db\tools;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Schema\AbstractAsset;
+use Doctrine\DBAL\Schema\Comparator;
+use Doctrine\DBAL\Schema\Index;
+use Doctrine\DBAL\Schema\Table;
+use phpbb\db\doctrine\table_helper;
+
+/**
+ * BC layer for database tools.
+ *
+ * In general it is recommended to use Doctrine directly instead of this class as this
+ * implementation is only a BC layer.
+ *
+ * In the 3.3.x version branch this class could return SQL statements instead of
+ * performing changes. This functionality has been removed.
+ */
+class doctrine implements tools_interface
+{
+	/**
+	 * @var Comparator
+	 */
+	private $comparator;
+
+	/**
+	 * @var \Doctrine\DBAL\Schema\AbstractSchemaManager
```

Comment:
use `use` here

Reply:
I don't think it makes sense, this is only used in the type hint here.